assets/app/data/crowbar/services/upgrade.factory.spec.js

```diff
@@ -9,13 +9,43 @@ describe('Upgrade Factory', function () {
         mockedCreateAdminBackupResponse = {
             id: 42
         },
+        mockedNewDatabaseServerResponse = {
+            'database_setup': {
+                'success': true
+            },
+            'database_migration': {
+                'success': true
+            },
+            'schema_migration': {
+                'success': true
+            },
+            'crowbar_init': {
+                'success': true
+            }
+        },
+        mockedConnectDatabaseServerResponse = {
```

Comment:
I would change it to `mockedConnectDatabaseServerResponse = mockedNewDatabaseServerResponse`. This gives us the effect of separation as you mentioned but also shows that at the moment the responses are identical.